packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -142,6 +164,7 @@ export class Popover implements ComponentInterface {
       placement: this.inoPlacement,
       trigger: this.inoTrigger,
       interactive: this.inoInteractive,
+      offset: [0, this.inoDistance],
```

Comment:
Cannot add a review for unchanged lines :( Another left over from the previous MR:
In l. 209 there is a second `ino-popover__content` which is a duplicate of the content a few lines above. In my opinion the second content could be removed.

Reply:
You mean this:
https://github.com/inovex/elements/blob/ce8569a1b5ef384d2dfb803133a7c924766e0c43/packages/elements/src/components/ino-popover/ino-popover.tsx#L185-L189

Should be replaced with:
```html
<div class="ino-tooltip__inner">
  <slot></slot>
</div>
```
?

Reply:
Done ✅